cpp/ps0007/cpp_ps0007_solution.cpp

```diff
@@ -6,12 +6,16 @@ float Area(char shape, float dimension[]);
 
 int main()
 {
+	// prompting user for the shape 1/1.5
 	char shape;
-	cout << "enter the shape \n circle -sqare-rectangle\n";
+	cout << "enter the first letter of the shape name \n c:circle \n s:square \n r:rectangle\n";
 	cin >> shape;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input
-0.5 cuz of your prompt. It wasn't so clear to the user